home/utils.py

```diff
@@ -11,7 +10,8 @@ def email(backend, response):
         return response['emails'][0]['value']
 
 
-def set_profile(backend, response, user, is_new=False, *args, **kwargs):    
+def set_profile(backend, response, user, is_new=False, *args, **kwargs):
+    from home.models import Profile    
```

Comment:
Why not keep all the import statements in top of the file?

Reply:
because of django initialize, when we use a imported functions on models is necessary to import on top of function to import.
You have any suggest to fix it ?

Reply:
I didn't know that. I can think nothing for now.